action.yml

```diff
@@ -21,8 +21,17 @@ inputs:
   previews:
     description: A comma-separated list of API previews to accept
   result-encoding:
-    description: Either "string" or "json" (default "json")—how the result will be encoded 
+    description: Either "string" or "json" (default "json")—how the result will be encoded
     default: json
+  retries:
+    description: "The number of times to retry a request"
+    default: "0"
+  retry-after:
+    description: "The number of seconds between retries. No effect unless `retries` is set."
+    default: "0"
+  do-not-retry:
+    description: "A comma separated list of status codes that will NOT be retried. Example: '400,500'. No effect unless `retries` is set."
```

Comment:
Should we mention that we'll default to the Octokit default if this is not set?

Reply:
👍